tools-local/unix/cmake/get-version.sh

```diff
@@ -2,36 +2,28 @@
 
 usage()
 {
-    echo "usage: $0 <repository-root> cmake <override-scripts-folder-path> <strict-tool-version-match> <tool-path>"
-    echo "repository-root                   This argument is ignored."
+    echo ""
+    echo "usage: $0 <repository-root> cmake <override-scripts-folder-path> <tool-path>"
+    echo "repository-root                   Path to repository root."
     echo "override-scripts-folder-path      This argument is ignored."
-    echo "strict-tool-version-match         This argument is ignored."
     echo "tool-path                         Path to CMake executable or the folder containing the executable."
     echo ""
     echo "If successful then, returns the version number of CMake executable."
     echo "Exit 1 if the version of the executable is an empty string."
+    echo ""
```

Comment:
> Why is there a trailing echo ""?

Readability.

> Seems inconsistent with other scripts.

`search-tool.sh` and `acquire-tool.sh` are missing trailing and leading `echo ""`.  I will update.